funcParser_tool/LICENSE

```diff
@@ -0,0 +1,72 @@
+GNU GENERAL PUBLIC LICENSE
+Version 2, June 1991
+
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.
```

Comment:
This should be the copyright date and name of author for *this* software

See the section "How to Apply These Terms to Your New Programs" in https://www.gnu.org/licenses/old-licenses/gpl-2.0-standalone.html

Reply:
Good catch! Updated.

Reply:
Sorry, that was wrong. This is the copyright statement for the copyright holder of the GNU GENERAL PUBLIC LICENSE text. I doubt that FSF has transfer the copyright of this license text to Intel.

Reply:
Restored to original statement.